### PR TITLE
fix(readme): Pre v0.6.0 Readme fixups

### PR DESCRIPTION
**Pull request by cpheps**
### Proposed Change
- Fixed spelling mistake
- Added macOS one liner to quickstart

##### Checklist
- [ ] Changes are tested
- [ ] CI has passed

### CODE REVIEW

**Review by BinaryFissionGames — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -54,9 +54,7 @@ For more installation information see [installing on Windows](/docs/installation
 To install the collector on macOS use the following brew commands:
```
Comment (StefanKurek) [inline README.md:54]: Should this be changed to no longer say "brew commands"?

Comment (cpheps) [reply README.md:54]: good catch

File: README.md [hunk 2]
```diff
@@ -51,12 +51,10 @@ For more installation information see [installing on Windows](/docs/installation
...
 #### macOS
 
-To install the collector on macOS use the following brew commands:
+To install using the installation script, you may run:
 
 ```sh
-brew tap observiq/homebrew-observiq-otel-collector
-brew update
-brew install observiq/observiq-otel-collector/observiq-otel-collector
+sh -c "$(curl -fsSlL https://github.com/observiq/observiq-otel-collector/releases/latest/download/install_macos.sh)" install_macos.sh
```
Comment (armstrmi) [inline README.md:57]: Could you explain real quick what this command does specifically, I've seen `sh` but I tried looking at the man pages for it and couldn't find the `-c` options I'm assuming it grabs the install file returned from that url posted and then installs it

**Review by StefanKurek — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (cpheps) [reply README.md:57]: I'll send you a link over slack for it.

**Pull request merged**